modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
+        logger.log('sending a ReceiverVideoConstraint message with '
```

Comment:
Would you need more code then to make sure the jitsi-meet UI can easily access this method through a public api? I'm not sure if at this point BridgeChannel is well exposed.